app/models/type_de_champ.rb

```diff
@@ -31,11 +31,14 @@ class TypeDeChamp < ApplicationRecord
   }
 
   belongs_to :procedure
+  belongs_to :procedure_revision
 
   belongs_to :parent, class_name: 'TypeDeChamp'
   has_many :types_de_champ, -> { ordered }, foreign_key: :parent_id, class_name: 'TypeDeChamp', inverse_of: :parent, dependent: :destroy
 
   store_accessor :options, :cadastres, :quartiers_prioritaires, :parcelles_agricoles, :old_pj, :drop_down_options
+  has_many :procedure_revision_types_de_champ, dependent: :destroy, inverse_of: :type_de_champ
```

Comment:
Pour faire plus court sur le nom, 

```suggestion
  has_many :revised_types_de_champ, dependent: :destroy, inverse_of: :type_de_champ
```

peut-être ? Ou ça te semble bizarre ?

Reply:
franchement je n'ai pas de bon nom pour ce modèle... mais pourquoi pas